.github/workflows/ci.yml

```diff
@@ -89,43 +89,29 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04]
-        containerd: [v1.6.21, v1.7.1]
+        containerd: [1.6.21, 1.7.1]
 
     steps:
       - name: Checkout extensions
         uses: actions/checkout@v4
 
+      - name: Download containerd release v${{ matrix.containerd }} archive
+        uses: robinraju/release-downloader@v1.8
+        with:
+          repository: "containerd/containerd"
+          tag: v${{ matrix.containerd }}
+          fileName: "containerd-${{ matrix.containerd }}-linux-amd64.tar.gz"
+          extract: true
```

Comment:
I think same can be achieved with gh CLI:

```
❯ gh release download --repo containerd/containerd --pattern '*linux-amd64.tar.gz' --output containerd.zip v1.7.3
```

Reply:
yes! Updated in https://github.com/containerd/rust-extensions/pull/203/commits/ee2a2330f5b3867b068c7194102c8f4d190bef09